app-editors/neovim/neovim-0.7.0.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8 work?

Reply:
I thought I fixed that as part of the bump! I'll raise a new PR to bump the EAPI this evening if it all tests OK.